lib/handlers/get.js

```diff
@@ -93,8 +93,8 @@ function handler (req, res, next) {
     // Translate from the contentType found to the possibleRDFType desired
     translate(stream, baseUri, contentType, possibleRDFType, function (err, data) {
       if (err) {
-        debug('error translating: ' + req.originalUrl + ' ' + contentType + ' -> ' + possibleRDFType + ' -- ' + 500 + err.message)
-        return next(error(500, err.message))
+        debug('error translating: ' + req.originalUrl + ' ' + contentType + ' -> ' + possibleRDFType + ' -- ' + 406 + ' ' + err.message)
+        return next(error(406, err.message))
```

Comment:
OK, I'll revert to 500 for now, at least until we add tests that prove otherwise.